internal/pkg/bulk/engine.go

```diff
@@ -115,6 +118,11 @@ func (b *Bulker) Client() *elasticsearch.Client {
 	return client
 }
 
+func (b *Bulker) Opts() BulkOpt {
+	return func(o *bulkOptT) {
```

Comment:
this is kind of a dummy function, I needed pass a BulkOpt to `NewBulker` [here](https://github.com/elastic/fleet-server/blob/e901a80e021aad672e716b14a59f1f9b39099bb0/internal/pkg/policy/policy_output.go#L287)

Reply:
Opts are optional, you should be able to call `NewBulker` with the transport and tracer